openqa-investigate

```diff
@@ -70,7 +70,7 @@ clone() {
     [[ -n ${*:5} ]] && clone_settings+=("${@:5}")
     # clear "PUBLISH_" settings to avoid overriding production assets
     # shellcheck disable=SC2207
-    clone_settings+=($(echo "$clone_job_data" | runjq -r '.job.settings | keys[] | select (startswith("PUBLISH_")) | . + "="')) || return $?
+    clone_settings+=($(echo "$clone_job_data" | runjq -r '.job.settings | keys[] | select (startswith("PUBLISH_")) | . + "=None"')) || return $?
```

Comment:
"none" is lower-case in English (maybe not in Python but I guess that doesn't matter here):
```suggestion
    clone_settings+=($(echo "$clone_job_data" | runjq -r '.job.settings | keys[] | select (startswith("PUBLISH_")) | . + "=none"')) || return $?
```

Reply:
Would you expect to be able to publish a disk called `None`? Maybe it should just be caseless anyway - ofc that needs to be implemented in os-autoinst, not here

Reply:
Yes, the matching in os-autoinst should probably be case-insensitive.

Reply:
https://github.com/os-autoinst/os-autoinst/pull/2470

Reply:
If you reference an issue like this:
* #os-autoinst/os-autoinst/pull/2470

then one sees the current status right away.

Reply:
how did you do that? :)

Reply:
In markdown list format with `#<id>`